Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -683,9 +678,10 @@ static LayoutRect sizingBox(RenderObject& renderer)
     return box.style().boxSizing() == BoxSizing::BorderBox ? box.borderBoxRect() : box.computedCSSContentBoxRect();
 }
 
-static Ref<CSSFunctionValue> matrixTransformValue(const TransformationMatrix& transform, const RenderStyle& style)
+static Ref<CSSFunctionValue> matrixTransformValue(TransformationMatrix transform, const RenderStyle& style)
 {
     RefPtr<CSSFunctionValue> transformValue;
+    transform.zoom(1 / style.effectiveZoom());
```

Comment:
This reciprocal is causing imported/w3c/web-platform-tests/css/css-transforms/animation/transform-interpolation-computed-value.html test failure. We should avoid this.